app/models/course_wiki_namespaces.rb

```diff
@@ -0,0 +1,3 @@
+class CourseWikiNamespaces < ApplicationRecord
```

Comment:
You should run `annotate` to generate the comments showing the columns of this table.